src/cursor.rs

```diff
@@ -96,6 +96,7 @@ impl Cursor {
     /// `doc` - Specifies the command that is being run.
     /// `cmd_type` - The type of command, which will be used for monitoring events.
     /// `read_pref` - The read preference for the query.
+    /// `batch_size` - The number of records to return per batch
```

Comment:
I'd add a note that specifies that the default is 1 to facilitate sending commands